### ISSUE TITLE
<Brush /> accessibility issues

### ISSUE THREAD

**Issue by enriquetamames-cpi** [recharts/recharts#3941]
- [ ] I have searched the [issues](https://github.com/recharts/recharts/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://master--63da8268a0da9970db6992aa.chromatic.com/?path=/story/examples-barchart--with-brush](https://master--63da8268a0da9970db6992aa.chromatic.com/?path=/story/examples-barchart--with-brush)

### Steps to reproduce
Go to tab "accessibility"

### What is expected?
No violations expected (2 violations found)

### What is actually happening?
For ".recharts-brush-traveller.recharts-layer[role="slider"]:nth-child(3)"

aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

For ".recharts-brush-traveller.recharts-layer[role="slider"]:nth-child4)"

aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Element has no title attribute

| Environment | Info |
|---|---|
| Recharts | v2.9.3 |
| React | v18.2.0 |
| System | Linux - Ubuntu |
| Browser | 112.0.5615.165 |

**Comment by oluSammy** [comment]
I have a similar issue with the brush
`ARIA input fields do not have accessible names`

`div.recharts-wrapper > svg.recharts-surface > g.recharts-layer > g.recharts-layer
<g class="recharts-layer recharts-brush-traveller" tabindex="0" role="slider" style="cursor: col-resize;">`

<img width="723" alt="Screenshot 2023-11-08 at 5 41 24 PM" src="https://github.com/recharts/recharts/assets/46409692/1ccedaba-67a0-4c66-bbcc-f6179e716c9d">

**Comment by ckifer** [comment]
Hey guys, this is known - recharts was not originally written with accessibility in mind and now we're slowly trying to improve. Going to take a while especially because we rely on community contributions. If you'd like to fix these issues please feel free to contribute - on mobile right now but I can link related issues and PRs

**Comment by ckifer** [comment]
We use forking - you should be able to create a PR from your fork

**Comment by ckifer** [comment]
Ty for vis, closing as completed
